### PR TITLE
Improve code action annotation UX

### PR DESCRIPTION
**Pull request by husanjun**

### CODE REVIEW

File: plugin/documents.py [hunk 1]
```diff
@@ -489,9 +490,13 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
...
             icon = 'Packages/LSP/icons/lightbulb.png'
         else:  # 'annotation'
-            suffix = 's' if action_count > 1 else ''
-            code_actions_link = make_command_link('lsp_code_actions', '{} code action{}'.format(action_count, suffix))
-            annotations = ["<div class=\"actions\">{}</div>".format(code_actions_link)]
+            if action_count > 1:
+                title = '{} code actions'.format(action_count)
+            else:
+                title = next(iter(responses.values()))[0]['title']
+                title = "<br/>".join(textwrap.wrap(title, width=30))
```
Comment (rchl) [inline plugin/documents.py:497]: `<br>` was never a self-closing element in the spec. `<br>` is enough. :)

Comment (husanjun) [reply plugin/documents.py:497]: Yes,  but in my test, it works and `\n` doesn't work

**Review by rwols — APPROVED** [review]
I like it :)

image

File: plugin/documents.py [hunk 2]
```diff
@@ -489,9 +490,13 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
             scope = 'region.yellowish lightbulb.lsp'
             icon = 'Packages/LSP/icons/lightbulb.png'
         else:  # 'annotation'
-            suffix = 's' if action_count > 1 else ''
-            code_actions_link = make_command_link('lsp_code_actions', '{} code action{}'.format(action_count, suffix))
-            annotations = ["<div class=\"actions\">{}</div>".format(code_actions_link)]
+            if action_count > 1:
+                title = '{} code actions'.format(action_count)
+            else:
+                title = next(iter(responses.values()))[0]['title']
```
Comment (rchl) [inline plugin/documents.py:496]: Something weird happened here. This is some old code that is wrong. This was supposed to be:
```py
title = next(itertools.chain.from_iterable(responses.values()))['title']
```

**Pull request merged**